tools/keycloak_setup/collection/roles/setup/vars/catalog_rbac.yml

```diff
@@ -0,0 +1,281 @@
+            description: 'Administrator access to all order items'
+            logic: POSITIVE
+            name: catalog-admin-all-order-item-permission
+            resourceType: catalog:order_item
+            type: resource
+          - decisionStrategy: UNANIMOUS
+            description: 'Administrator access to all progress messages'
+            logic: POSITIVE
+            name: catalog-admin-all-progress-message-permission
+            resourceType: catalog:progress_message
+            type: resource
+        type: role
+    resources:
+      - displayName: 'All Portfolios'
+        name: catalog:portfolio:all
+        scopes:
+          - displayName: 'Catalog Portfolio Create'
+            name: catalog:portfolio:create
+          - displayName: 'Catalog Portfolio Read'
+            name: catalog:portfolio:read
+          - displayName: 'Catalog Portfolio Update'
+            name: catalog:portfolio:update
+          - displayName: 'Catalog Portfolio Delete'
+            name: catalog:portfolio:delete
+          - displayName: 'Catalog Portfolio Order'
+            name: catalog:portfolio:order
+        type: catalog:portfolio
+        uris:
+          - api/v1/portfolios/*
+      - displayName: 'All Portfolio Items'
+        name: catalog:portfolio_item:all
+        scopes:
+          - displayName: 'Catalog Portfolio Item Create'
+            name: catalog:portfolio_item:create
+          - displayName: 'Catalog Portfolio Item Read'
+            name: catalog:portfolio_item:read
+          - displayName: 'Catalog Portfolio Item Update'
+            name: catalog:portfolio_item:update
+          - displayName: 'Catalog Portfolio Item Delete'
+            name: catalog:portfolio_item:delete
+          - displayName: 'Catalog Portfolio Item Order'
+            name: catalog:portfolio_item:order
+        type: catalog:portfolio_item
+        uris:
+          - api/v1/portfolio_items/*
+      - displayName: 'All Service Plans'
+        name: catalog:service_plan:all
+        scopes:
+          - displayName: 'Catalog Service Plan Create'
+            name: catalog:service_plan:create
+          - displayName: 'Catalog Service Plan Read'
+            name: catalog:service_plan:read
+          - displayName: 'Catalog Service Plan Update'
+            name: catalog:service_plan:update
+          - displayName: 'Catalog Service Plan Delete'
+            name: catalog:service_plan:delete
+        type: catalog:service_plan
+        uris:
+          - api/v1/service_plans/*
+      - displayName: 'All Orders'
+        name: catalog:order:all
+        scopes:
+          - displayName: 'Catalog Order Create'
+            name: catalog:order:create
+          - displayName: 'Catalog Order Read'
+            name: catalog:order:read
+          - displayName: 'Catalog Order Update'
+            name: catalog:order:update
+          - displayName: 'Catalog Order Delete'
+            name: catalog:order:delete
```

Comment:
@Alex-Izquierdo This is the typo fix for https://issues.redhat.com/browse/SSP-2811